deploy/sre-prometheus/100-managed-velero-operator.PrometheusRule.yaml

```diff
@@ -23,7 +23,7 @@ spec:
         scalar(max(kube_deployment_created{namespace="openshift-velero",deployment="velero"}))) > 10800 + 600
       labels:
         severity: warning
-        namespace: openshift-monitoring
+        namespace: '{{ $labels.namespace }}'
```

Comment:
I believe this should be double quotes not single quotes

Reply:
```
rg -i 'labels.namesp' | grep deploy                                                                                        ✔  22  
deploy/sre-prometheus/100-csv-abnormal.PrometheusRule.yaml:        namespace: "{{ $labels.namespace }}"
deploy/sre-prometheus/100-csv-abnormal.PrometheusRule.yaml:        namespace: "{{ $labels.namespace }}"
deploy/sre-prometheus/100-managed-velero-operator.PrometheusRule.yaml:        namespace: "{{ $labels.namespace }}"
deploy/sre-prometheus/100-managed-velero-operator.PrometheusRule.yaml:        namespace: "{{ $labels.namespace }}"
deploy/sre-prometheus/100-managed-velero-operator.PrometheusRule.yaml:        namespace: "{{ $labels.namespace }}"
deploy/sre-prometheus/100-excessive-memory.PrometheusRule.yaml:        namespace: "{{ $labels.namespace }}"
deploy/sre-prometheus/100-excessive-memory.PrometheusRule.yaml:        message: System container {{ $labels.namespace }}/{{ $labels.pod_name }}/{{ $labels.container_name }}
deploy/sre-prometheus/100-excessive-memory.PrometheusRule.yaml:        namespace: "{{ $labels.namespace }}"
deploy/sre-prometheus/100-excessive-memory.PrometheusRule.yaml:        message: System container {{ $labels.namespace }}/{{ $labels.pod_name }}/{{ $labels.container_name }}
deploy/sre-prometheus/100-sre-kubequotaexceeded.PrometheusRule.yaml:        message: Quota limit exceeded in namespace {{ $labels.namespace }}.
deploy/sre-prometheus/100-elasticsearch-jobs.PrometheusRule.yaml:        namespace: '{{ $labels.namespace }}'
deploy/sre-prometheus/100-elasticsearch-jobs.PrometheusRule.yaml:        message: Elasticsearch job {{ $labels.job_name }} has failed 2 or more times in the last 15 minutes in namespace {{ $labels.namespace }}.
deploy/sre-prometheus/100-sre-pruning.PrometheusRule.yaml:        message: SRE Pruning Job {{ $labels.namespace }}/{{ $labels.cronjob }}
deploy/sre-prometheus/100-haproxy-reload-fails.Prometheusrule.yaml:        namespace: "{{ $labels.namespace }}"
deploy/sre-prometheus/100-sre-pv.PrometheusRule.yaml:        namespace: "{{ $labels.namespace }}"
deploy/sre-prometheus/100-sre-pv.PrometheusRule.yaml:        message: "The customer PersistentVolume claimed by {{ $labels.persistentvolumeclaim }} in Namespace {{ $labels.namespace }} is only {{ printf \"%0.2f\" $value }}% free."
deploy/sre-prometheus/100-sre-pv.PrometheusRule.yaml:        namespace: "{{ $labels.namespace }}"
deploy/sre-prometheus/100-sre-pv.PrometheusRule.yaml:        message: "Based on recent sampling, the customer PersistentVolume claimed by {{ $labels.persistentvolumeclaim }} in Namespace {{ $labels.namespace }} is expected to fill up within four days. Currently {{ printf \"%0.2f\" $value }}% is available."
deploy/sre-prometheus/100-sre-pv.PrometheusRule.yaml:        namespace: "{{ $labels.namespace }}"
deploy/sre-prometheus/100-sre-pv.PrometheusRule.yaml:        message: "The customer PersistentVolume claimed by {{ $labels.persistentvolumeclaim }} in Namespace {{ $labels.namespace }} is only {{ printf \"%0.2f\" $value }}% free."
deploy/sre-prometheus/100-sre-pv.PrometheusRule.yaml:        namespace: "{{ $labels.namespace }}"
deploy/sre-prometheus/100-sre-pv.PrometheusRule.yaml:        message: "Based on recent sampling, the customer PersistentVolume claimed by {{ $labels.persistentvolumeclaim }} in Namespace {{ $labels.namespace }} is expected to fill up within four days. Currently {{ printf \"%0.2f\" $value }}% is available."
```

Looks like we do ', " and sometimes none. I updated the above to use "" and it generated no difference in the outputs of `/hack`